scripts/mbedtls_dev/c_build_helper.py

```diff
@@ -45,9 +45,11 @@ def create_c_file(file_label):
                                     suffix='.c')
     exe_suffix = '.exe' if platform.system() == 'Windows' else ''
     exe_name = c_name[:-2] + exe_suffix
+    obj_name = c_name[:-2] + '.obj'
     remove_file_if_exists(exe_name)
+    remove_file_if_exists(obj_name)
     c_file = os.fdopen(c_fd, 'w', encoding='ascii')
-    return c_file, c_name, exe_name
+    return c_file, c_name, exe_name, obj_name
```

Comment:
How about not bothering with `obj_name` here and using `exe_name[:-3]obj` in the caller?